docs/index.html

```diff
@@ -1353,6 +1353,27 @@ <h4 class="headline-3">Module: Button</h4>
 		</div>
 	</section>
 
+	<section class="grid">
+		<div class="grid-row">
+			<div class="grid-cell x3 hint-text">
+				<h4 class="headline-3">Module: Issue Labels</h4>
+				<p>
+					This Module helps you to set filter labels. The ID <code>filter-issue-status</code> wraps the labels. The <code>filter-label</code> class makes sure, that all labels are wrapped in a container with a 100% width and the text is centered. The current used label gets the class <code>active</code> as well to show in wich state the ticket is currently. 
```

Comment:
This module helps to add a label to the issue, which indicates the status of an issue. 
There are 6 different states.

- Needs Triage 
- Needs Diagnosis
- Needs Contact
- Ready for Outreach
- Site Contacted
- Closed